ui/src/app/utils/runtime-utils.tsx

```diff
@@ -356,13 +401,19 @@ export const useRuntimeStatus = (currentWorkspaceNamespace, currentGoogleProject
 // all runtime state.
 export const useCustomRuntime = (currentWorkspaceNamespace):
     [{currentRuntime: Runtime, pendingRuntime: Runtime}, (runtime: Runtime) => void] => {
```

Comment:
It might be cleaner to instead add a 3rd value here, instead of direct store interactions from the panel: `reattachableDisk`